bots/notify/src/main/java/org/openjdk/skara/bots/notify/issue/JbsBackport.java

```diff
@@ -50,6 +52,7 @@ private static URI backportRequest(URI uri) {
     private Issue createBackportIssue(Issue primary) {
         var finalProperties = new HashMap<>(primary.properties());
         finalProperties.put("issuetype", JSON.of("Backport"));
+        finalProperties.remove(RESOLVED_IN_BUILD);
```

Comment:
Minor: did you mean to make this change in this PR? It doesn't seem behavior neutral (might be related to the follow-on PR?)

Reply:
Removed from this change and applied in the follow-on.